src/main/java/org/broadinstitute/consent/http/models/Study.java

```diff
@@ -79,14 +78,11 @@ public Set<StudyProperty> getProperties() {
     return properties;
   }
 
-  public void setProperties(Set<StudyProperty> properties) {
-    this.properties = properties;
+  public void addProperties(Set<StudyProperty> properties) {
+    this.properties.addAll(properties);
   }
```

Comment:
Given how this is used, you could make it easier on the caller and accept varargs
```suggestion
  public void addProperties(StudyProperty... properties) {
    this.properties.addAll(Set.of(properties));
  }
```